tests/providers/amazon/aws/operators/test_redshift_cluster.py

```diff
@@ -76,6 +77,10 @@ def test_create_single_node_cluster(self, mock_get_conn):
             **params,
         )
 
+        mock_get_conn.return_value.get_waiter.return_value.wait.assert_called_once_with(
```

Comment:
We should test that the waiter is not called when `wait_for_completion` is `False`

Reply:
Added the test when wait_for_completion is False and I think it makes sense to have this test too.